### PR TITLE
TRM-22119 add proteincount, json confif and builder for disease model

### PR DESCRIPTION
**Pull request by sahmad-ebi**
@LeonardoGonzales ,
Please review this as well. Part of same Jira TRM-22119

### CODE REVIEW

File: json-parser/src/test/java/uk/ac/ebi/uniprot/json/parser/disease/DiseaseTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package uk.ac.ebi.uniprot.json.parser.disease;
```
Comment (LeonardoGonzales) [inline json-parser/src/test/java/uk/ac/ebi/uniprot/json/parser/disease/DiseaseTest.java]: Add one more test with minimum fields
